packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/ClusterItemListener.java

```diff
@@ -0,0 +1,11 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
This method combines both:
ClusterManagersController.OnClusterItemMarker<MarkerBuilder>,
and 
ClusterManager.OnClusterItemClickListener<MarkerBuilder>
under same listener.

Listener with type ClusterItemListener is registered for clusterManagerController.
It is possible to have two listenerer objects instead of this combinations, and two setter functions to set the listener.

Instead of method   
```
void setClusterItemListener(@Nullable ClusterItemListener listener) {
    clusterItemListener = listener;
    initListenersForClusterManagers(this, listener);
}
```

there need to be two methods:

```
  void setOnClusterItemClickListener(@Nullable ClusterManager.OnClusterItemClickListener<MarkerBuilder> listener) {
    onClusterItemClickListener= listener;
    initListenersForClusterManagers(this, listener);
  }
```
  
  and
  
```
  void setOnClusterItemRenderedListener(@Nullable ClusterManagersController.OnClusterItemMarker<MarkerBuilder>listener) {
    onClusterItemRenderedListener = listener;
  }
```

Is this approach better?